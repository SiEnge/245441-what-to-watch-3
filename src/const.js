export const MINUTE_IN_ONE_HOUR = 60;
export const TEN_MINUTES = 10;

export const AuthorizationStatus = {
  AUTH: `AUTH`,
  NO_AUTH: `NO_AUTH`
};

export const AppRoute = {
  LOGIN: `/login`,
  REVIEW: `/review`,
  MY_LIST: `/mylist`,
  FILMS: `/films`,
  PLAYER: `/player`,
  ROOT: `/`,
};

export const TabsType = {
  OVERVIEW: `Overview`,
  DETAIL: `Details`,
  REVIEWS: `Reviews`
};

export const ScoreType = {
  BAD: `Bad`,
  NORMAL: `Normal`,
  GOOD: `Good`,
  VERY_GOOD: `Very good`,
  AWESOME: `Awesome`,
};

// const GenreType = {
//   ALL_GENRES: `All genres`,
//   COMEDIAS: `Comedies`,
//   CRIME: `Crime`,
//   DOCUMENTARY: `Documentary`,
//   DRAMAS: `Dramas`,
//   HORROR: `Horror`,
//   ROMANCE: `Romance`,
//   SCI_FI: `Sci-Fi`,
//   THRILLERS: `Thrillers`,
// };
