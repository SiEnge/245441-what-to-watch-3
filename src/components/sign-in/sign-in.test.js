import React from "react";
import renderer from "react-test-renderer";
import {Router} from "react-router-dom";
import SignIn from "./sign-in.jsx";
import history from "../../history.js";
import {noop} from "../../const.js";

it(`Render SignIn`, () => {
  const tree = renderer
    .create(
        <Router history={history}>
          <SignIn
            onSubmit={noop}
          />
        </Router>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
