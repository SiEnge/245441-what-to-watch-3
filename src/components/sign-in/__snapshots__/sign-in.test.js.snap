// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render SignIn 1`] = `
<div
  className="user-page"
>
  <header
    className="page-header user-page__head"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link"
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <h1
      className="page-title user-page__title"
    >
      Sign in
    </h1>
  </header>
  <div
    className="sign-in user-page__content"
  >
    <form
      action="#"
      className="sign-in__form"
      onSubmit={[Function]}
    >
      <div
        className="sign-in__fields"
      >
        <div
          className="sign-in__field"
        >
          <input
            className="sign-in__input"
            id="user-email"
            name="user-email"
            placeholder="Email address"
            type="email"
          />
          <label
            className="sign-in__label visually-hidden"
            htmlFor="user-email"
          >
            Email address
          </label>
        </div>
        <div
          className="sign-in__field"
        >
          <input
            className="sign-in__input"
            id="user-password"
            name="user-password"
            placeholder="Password"
            type="password"
          />
          <label
            className="sign-in__label visually-hidden"
            htmlFor="user-password"
          >
            Password
          </label>
        </div>
      </div>
      <div
        className="sign-in__submit"
      >
        <button
          className="sign-in__btn"
          type="submit"
        >
          Sign in
        </button>
      </div>
    </form>
  </div>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
        href="/"
        onClick={[Function]}
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        Â© 2019 What to watch Ltd.
      </p>
    </div>
  </footer>
</div>
`;
